.env.development

```diff
@@ -57,3 +57,8 @@ OTEL_SDK_DISABLED=false
 #EMAIL_SMTP_ALLOW_INVALID_CERT=
 #EMAIL_SMTP_SENDER_NAME=
 #EMAIL_SMTP_SENDER_EMAIL=
+
+HEADLESS_BROWSER_HOST=headless-browser
```

Comment:
You still need to run headless browser in a separate docker container..

Reply:
that's ok, it could be confusing for some people because the error is less clear if you are not running headless_browser. But we can merge as it is